home3.2/src/main/java/Triangle.java

```diff
@@ -0,0 +1,36 @@
+public class Triangle {
+    private double side1;
+    private double side2;
+    private double side3;
+
+    public Triangle(double side1, double side2, double side3) {
+        this.side1 = side1;
+        this.side2 = side2;
+        this.side3 = side3;
+    }
+
+    public double calculateArea() {
+        double s = (side1 + side2 + side3) / 2.0;
+        return Math.sqrt(s * (s - side1) * (s - side2) * (s - side3));
+    }
+
+    public double getHeight(char side) {
```

Comment:
it is better to have enum type. As a result conversion from user input will be separated from this class and it will handle only correct cases for side